doc/source/user/building.rst

```diff
@@ -152,16 +153,25 @@ list is retained.
 One cannot mix negation and positives, nor have multiple negations, such cases will
 raise an error.
 
+The generic option expects the environment to provide a BLAS_LIBS
+variable that contains all linker flags to link to a standard BLAS
+library that offers the Fortran interface (cblas symbols not assumed
+to be present). You can use this mechanism to ensure that the build is
+agnostic to the actual BLAS implementation. This is also the first in
+the list to enable the not uncommon behaviour of using BLAS_LIBS from
+the environment and only guessing if it is not set.
+
 LAPACK
 ~~~~~~
 
 The default order for the libraries are:
 
-1. MKL
-2. OpenBLAS
-3. libFLAME
-4. ATLAS
-5. LAPACK (NetLIB)
+1. generic (LAPACK_LIBS from environment)
```

Comment:
The use case is pretty specific and bypasses a bunch of checks, so I'd rather say that if the env var is set, that value is used and no auto-detection of any library is attempted.

Reply:
Do you mean to just state that more clearly in the documentation?

Reply:
OK, I added a phrase to note that automatics are bypassed, but also added a prominent note that the CBLAS situation needs to be settled before merging.

Reply:
yes indeed